db/migrate/20220210093754_add_company_value_to_kudos.rb

```diff
@@ -0,0 +1,5 @@
+class AddCompanyValueToKudos < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :kudos, :company_value, null: true, foreign_key: true
```

Comment:
According to AC:
`NOT_NULL` constraint is set on `company_value_id` from `kudos` table
so I think you should have `null: false` here